logging/logging.go

```diff
@@ -29,7 +29,7 @@ func NewNilLogger() SimpleLogger {
 //--------------------------------------------------------------------
 type LoggerWithPrefix struct {
 	prefix        string
-	wrappedLogger SimpleLogger
+	WrappedLogger SimpleLogger
```

Comment:
this works, but might be clearer to keep the field private and add a public accessor like `WrappedLogger()`.. unless someone needs to dynamically change the value?